website/guides/05_rabbitmq.mdx

```diff
@@ -0,0 +1,473 @@
+
+```
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  annotations:
+    kubectl.kubernetes.io/last-applied-configuration: |
+      {"apiVersion":"rabbitmq.com/v1beta1","kind":"RabbitmqCluster","metadata":{"annotations":{},"name":"definition","namespace":"rabbitmq-system"}}
+  creationTimestamp: "2024-05-07T17:04:53Z"
+  finalizers:
+  - deletion.finalizers.rabbitmqclusters.rabbitmq.com
+  generation: 6
+  name: definition
+  namespace: rabbitmq-system
+  resourceVersion: "2729957"
+  uid: f9b4cad8-68b4-489e-b7a1-baa3b91882d5
+spec:
+  delayStartSeconds: 30
+  image: rabbitmq:3.13.1-management
+  override: {}
+  persistence:
+    storage: 10Gi
+  rabbitmq:
+    additionalConfig: |
+      load_definitions = /etc/rabbitmq/definitions.json
+  replicas: 3 # added three replicas which will for the RabbitMQ cluster
+  resources: 
+    limits:
+      cpu: "2"
+      memory: 2Gi
+    requests:
+      cpu: "1"
+      memory: 2Gi
+  secretBackend:
+    externalSecret: {}
+  service:
+    type: ClusterIP
+  terminationGracePeriodSeconds: 604800
+...
+```
+
+![RabbitMQ cluster view in dashboard](https://github.com/glasskube/glasskube/assets/38757612/231868a4-2c14-490d-b164-a9417f5c6344)
+
+Not only have the nodes been added to the cluster but the queue has been automatically replicated to all cluster nodes too
+
+![Queue replication](https://github.com/glasskube/glasskube/assets/38757612/5d1520ec-3e70-409d-8c66-a7873485e86a)
+
+Messaging brokers such as RabbitMQ play a central role in [event-driven architectures](https://en.wikipedia.org/wiki/Event-driven_architecture), offering `flexibility` for `task processing`, `workflow orchestration`, and `service integrations` while offering a selection of `messaging patterns` to choose from to best suit your architecture. An obvious choice when prioritizing `delivery reliability` and `fault tolerance`, due to its native acknowledgment and message persistence features.
+
+Getting started with RabbitMQ is easier than ever especially now that [Glasskube](https://github.com/glasskube/glasskube) integrates the cluster controller for easy installation and management. This guide aims to offer the necessary insights for setting up a functional instance or RabbitMQ cluster within a Kubernetes environment. While there's so much more to learn regarding RabbitMQ such as this and that, I hope this primer provides ample groundwork to kickstart your RabbitMQ journey.
+
+
+If you like this sort of content and would like to see more of it, please consider supporting us by giving us a Star on [GitHub(https://github.com/glasskube/glasskube)] 🙏
+[![Screenshot 2024-04-26 at 10 54 54](https://github.com/glasskube/glasskube/assets/38757612/05a03edf-7d6a-48fd-9b24-598abcfd1e9a)](https://github.com/glasskube/glasskube)
```

Comment:
Also looks a bit broken to me: 
![Screenshot from 2024-05-14 10-03-58](https://github.com/glasskube/glasskube/assets/7556574/a5c09f41-b08b-4ca1-ae4f-1e35bc3929a1)